src/panels/lovelace/editor/card-editor/hui-dialog-edit-card.ts

```diff
@@ -182,12 +164,9 @@ export class HuiDialogEditCard
         { type: cardName }
       );
     } else if (!this._cardConfig) {
-      heading = this._containerConfig.title
-        ? this.hass!.localize(
```

Comment:
Dont we need this anymore?

Reply:
It will never be used as the card config is required for the edit card now.